RealmSwift/EmbeddedObject.swift

```diff
@@ -95,11 +95,6 @@ extension EmbeddedObject: RealmCollectionValue {
         return nil
     }
 
-    /// The object schema which lists the managed properties for the object.
-    public var objectSchema: ObjectSchema {
-        return ObjectSchema(RLMObjectBaseObjectSchema(self)!)
-    }
```

Comment:
It now lives in the extension for `ObjectBase`, both `Object` and `EmbeddedObject` reference it from there now

Reply:
I think that may cause issues for the generated docs. There's a bunch of stuff redeclared in Object which don't strictly need to be because it's the only way to make jazzy put the docs on the right page.

Reply:
reverted the change